core/trino-spi/src/test/java/io/trino/spi/predicate/TestEquatableValueSet.java

```diff
@@ -295,32 +296,36 @@ public void testSubtract()
         assertEquals(EquatableValueSet.of(ID, 0L).complement().subtract(EquatableValueSet.of(ID, 0L, 1L).complement()), EquatableValueSet.of(ID, 1L));
     }
 
-    @Test(expectedExceptions = UnsupportedOperationException.class)
+    @Test
     public void testUnmodifiableCollection()
     {
-        EquatableValueSet.of(ID, 1L).getValues().clear();
+        assertThatThrownBy(() -> EquatableValueSet.of(ID, 1L).getValues().clear())
```

Comment:
i missed this one -- please  pull `EquatableValueSet.of(ID, 1L).getValues()` before  `assertThatThrownBy` , so this line becomes:

```
assertThatThrownBy(values::clear)
```